solutions/ha-nva/layer-7-ingress/azuredeploy.json

```diff
@@ -0,0 +1,801 @@
+            "defaultValue": "Yes",
+            "type": "String",
+            "allowedValues": [
+                "No",
+                "Yes"
+            ],
+            "metadata": {
+                "description": "Selection to deploy Azure Application Gateway for the Web"
+            }
+        },
+        "_artifactsLocation": {
+            "type": "string",
+            "metadata": {
+                "description": "The base URI where artifacts required by this template are located including a trailing '/'"
+            },
+            "defaultValue": "https://raw.githubusercontent.com/mspnp/samples/master/solutions/ha-nva/layer-7-ingress/"
+        },
+        "_artifactsLocationSasToken": {
+            "type": "securestring",
+            "metadata": {
+                "description": "The sasToken required to access _artifactsLocation"
+            },
+            "defaultValue": ""
+        }
+    },
+    "variables": {
+        "ostag": "[if(equals(parameters('OperatingSystemImage'),'Red Hat Enterprise Linux 7 (latest, LVM)'),'RH7x','Cent7x')]",
+        "osProfile": {
+            "RH7x": {
+                "image": {
+                    "publisher": "RedHat",
+                    "offer": "RHEL",
+                    "sku": "7-LVM",
+                    "version": "latest"
+                }
+            },
+            "Cent7x": {
+                "image": {
+                    "offer": "CentOS-LVM",
+                    "publisher": "OpenLogic",
+                    "sku": "7-LVM",
+                    "version": "latest"
+                }
+            }
+        },
+        "postInstallActions": {
+            "web": {
+                "commandToExecute": "sh simpleweb.sh",
+                "fileUris": [
+                    "[uri(parameters('_artifactsLocation'), concat('scripts/simpleweb.sh', parameters('_artifactsLocationSasToken')))]"
+                ]
+            },
+            "nva": {
+                "commandToExecute": "sh simpleproxy.sh",
+                "fileUris": [
+                    "[uri(parameters('_artifactsLocation'), concat('scripts/simpleproxy.sh', parameters('_artifactsLocationSasToken')))]"
+                ]
+            }
+        }
+    },
+    "resources": [
+        {
+            "type": "Microsoft.Network/networkSecurityGroups",
+            "apiVersion": "2019-12-01",
+            "name": "[parameters('WebNetworkSecurityGroupName')]",
+            "location": "[parameters('location')]",
+            "properties": {
+                "securityRules": [
+                    {
+                        "name": "AllowHTTPfromDMZtoWeb",
+                        "type": "Microsoft.Network/networkSecurityGroups/securityRules",
+                        "properties": {
+                            "protocol": "TCP",
+                            "sourcePortRange": "*",
+                            "destinationPortRange": "80",
+                            "sourceAddressPrefix": "10.0.0.64/27",
```

Comment:
Consider using parameters for source and destination prefixes both in NSG and Subnet?

Reply:
Good idea, I had forgotten that these were all hardcoded.

Reply:
These should all be parameterized now